lib/robots/dor_repo/dissemination/cleanup.rb

```diff
@@ -3,15 +3,16 @@
 module Robots
   module DorRepo
     module Dissemination
-      # NOTE: This step has been merged into the end accession robot. It should no longer be used, but is left in case
-      # there are existing workflow steps that require it.
+      # This step has been merged into the end accession robot. It should no longer be used.
       class Cleanup < LyberCore::Robot
         def initialize
           super('disseminationWF', 'cleanup')
         end
 
         def perform_work
           object_client.workspace.cleanup(workflow: 'disseminationWF', lane_id:)
+          # Oct 3 2024:  Warn. If we never see these warnings, we can get rid of this robot completely.
+          Honeybadger.notify('[WARNING] DisseminationWF:cleanup robot is deprecated and should not be used.', context: { druid: })
```

Comment:
we do not think this robot is every invoked - let's add an HB alert, if we don't see that alert ever, we can just ditch this whole robot